### PR TITLE
add new IH subpages, update main.lang with new footer strings

### PR DESCRIPTION
**Pull request by peiying2**
PR 4859 - IH hub
PR 4876 - unified footer

### CODE REVIEW

**Comment by peiying2** [comment]
@flodolo 
For PR 4876 - unified footer, I followed your instruction and checked only the site-footer.html against main.lang. There is a site-footer-new.html but it doesn't seem to have the new strings, which kind of puzzled me.  Anyway, I hope I got everything.  

For PR 4859 - IH hub: there are several smaller size files along the two bigger files, index and podcast.  This is what I have noticed:
- the "includes" folder and files don't seem to work on the test server, so I can't verify anything how it looks.  I am not sure if these smaller files should be standalone of consolidated.  
- lots of duplicate strings between the IH page/subpages, so there will lots of copy of strings once the files are added to locales and merged.  I wish there is a way to create a IH-main file for this set of files.  

I will not work on langchecker until we sort this part out.  Thanks!  Will be up early in the morning to work on this.  Thanks a mil!

File: en-US/main.lang [hunk 1]
```diff
@@ -657,3 +657,43 @@ Thanks for choosing Firefox!
 # Used in combination with other strings to create HTML page titles, e.g. in What's new pages
 ;Mozilla Firefox Web Browser
 Mozilla Firefox Web Browser
+
+
+# A social link in the footer
+;Twitter
```
Comment (flodolo) [inline en-US/main.lang]: Drop all social network names: Twitter, Facebook, Instagram, YouTube. We don't want them localized or transliterated (they were also in the current footer)

File: en-US/mozorg/internet-health/includes/newsletter.lang [hunk 2]
```diff
@@ -0,0 +1,10 @@
+## NOTE: Demo server is available at https://www-dev.allizom.org/internet-health/includes/newsletter/
+## URL: https://www-dev.allizom.org/%LOCALE%/internet-health/includes/newsletter/
+
+
+;Stay informed
+Stay informed
```
Comment (flodolo) [inline en-US/mozorg/internet-health/includes/newsletter.lang]: Drop this file, strings are already covered in the other files in /internet-health

File: en-US/mozorg/internet-health/includes/podcast.lang [hunk 3]
```diff
@@ -0,0 +1,20 @@
+## NOTE: Demo server is available at https://www-dev.allizom.org/internet-health/includes/podcast/
+## URL: https://www-dev.allizom.org/%LOCALE%/internet-health/includes/podcast/
```
Comment (flodolo) [inline en-US/mozorg/internet-health/includes/podcast.lang]: Include pages are not reachable, they're fragments of pages included into others.

**Comment by flodolo** [comment]
This is blocked by https://github.com/mozilla/bedrock/pull/4886

Langchecker bits: https://github.com/mozilla-l10n/langchecker/pull/660

**Pull request merged**